packages/block-editor/src/components/inserter/search-results.js

```diff
@@ -70,6 +71,9 @@ function InserterSearchResults( {
 		destinationRootClientId
 	);
 
+	const parentBlockListSettings = getBlockListSettings( rootClientId );
+	const parentInserterPriority = parentBlockListSettings?.inserterPriority;
```

Comment:
I think we _might_ need to compute this value inside a `useSelect` hook. Why? Because currently if the block list or `rootClientId` were to change then this component would not update. `useSelect` would re-run the selectors if dependencies change so we might need to refactor toward that.